src/background/dnr.js

```diff
@@ -20,7 +21,7 @@ if (__PLATFORM__ === 'chromium' || __PLATFORM__ === 'safari') {
   // Ensure that DNR rulesets are equal to those from options.
   // eg. when web extension updates, the rulesets are reset
   // to the value from the manifest.
-  observe(async (options) => {
+  observer.addListener(async function dnr(options) {
```

Comment:
why not the arrow function like in other places?

Reply:
I added logs for each observer, and it uses the property name or function name to log, which observer is running.